ush/wrappers/job_cards/qsub/run_post.qsub

```diff
@@ -0,0 +1,27 @@
+#! /bin/sh
+#PBS -A epic
+#PBS -j oe -o run_post.log
+#PBS -l select=2:mpiprocs=24:ncpus=24
+#PBS -q main
+#PBS -l walltime=00:15:00
+#PBS -N run_post
+
+#--- for propper work, make sure of placement of your job card relative to var_defns.sh file ---#
+export GLOBAL_VAR_DEFNS_FP=$PWD'/../var_defns.sh'
+export USHdir=`grep USHdir $GLOBAL_VAR_DEFNS_FP | cut -d\' -f2`
+export JOBSdir=`grep JOBSdir $GLOBAL_VAR_DEFNS_FP | cut -d\' -f2`
+export PDY=`grep DATE_FIRST_CYCL $GLOBAL_VAR_DEFNS_FP | cut -d\' -f2 | cut -c1-8`
+export cyc=`grep DATE_FIRST_CYCL $GLOBAL_VAR_DEFNS_FP | cut -d\' -f2 | cut -c9-10`
+export SLASH_ENSMEM_SUBDIR='/'
+export ENSMEM_INDX='000'
+export nprocs='48'
+
+export FCST_LEN_HRS='12'
```

Comment:
`FCST_LEN_HRS` is in the `var_defns.sh` that is exported earlier in the script. I'd remove this to avoid conflict since the forecast hour variable can vary for each experiment.